forte/data/ontology/code_generation_objects.py

```diff
@@ -622,6 +626,10 @@ def make_module_dirs(
               placed
             include_init: True if `__init__.py` is to be generated in existing
               packages in which `__init__.py` does not already exists
+            use_name_space_packaging: True if namespace packaging is to be used,
+                and `__init__.py` is to be generated at the top level;
+                False, namespace packaging is to be used,
```

Comment:
docstring might be wrong, what I read here is not matter True of False, namespace packaging is to be used.

Just to clarify, when we use name_space_packaging, we don't generate at the top level